Packs/ProofpointIsolation/README.md

```diff
@@ -0,0 +1,16 @@
+### Proofpoint Isolation
+Proofpoint Isolation is a fully managed cloud-based solution that provides secure web browsing by isolating user sessions, protecting against threats like malware, phishing, and data breaches.
+
+### Proofpoint Isolation Event Collector:
+To enable the Proofpoint Isolation Event Collector, follow these steps:
+1. Go to **Settings** &rarr; **Configurations** &rarr; **Automation & Feed Integrations**.
+2. In the search bar, type **Proofpoint Isolation**.
+3. At the right-corner, click **+ Add instance**.
+4. Follow the instructions in the prompt window to configure the Proofpoint Isolation Event Collector.
+
+
+#### How to Access Reporting API
+1. In Proofpoint Isolation, navigate to **Product Settings** > **Reporting API**.  Proofpoint Isolation’s Reporting API tools and documentation display in the Console’s main viewing panel.
+2. Copy the reporting API key.
+
+[image]("../../doc_files/readme_image.png")
```

Comment:
I don't think we need this part here. In the pack README should be general info and use-cases of the pack. 
We put how to configure in the integration readme.